compiler/src/main.sk

```diff
@@ -69,7 +69,13 @@ fun main(): void {
     _context ~> {
       SKStore.CStop(
         Some((context, _, _) ~> {
-          _result = compile(config, context, config.unknown);
+          try {
+            compile(config, context, config.unknown)
+          } catch {
+          | exn ->
+            print_error(exn.getMessage());
+            skipExit(1)
```

Comment:
What do you think about a special exit code to distinguish between "your code is faulty" and "the compiler is broken"?